src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -365,6 +365,10 @@ private static byte[] GetSqlServerSPN(string hostNameOrAddress, string portOrIns
             {
                 serverSpn += ":" + portOrInstanceName;
             }
+            else
+            {
+                serverSpn += $":{DefaultSqlServerPort}";
+            }
```

Comment:
Did you check it with an instance name?
Personally, I prefer to manage it at `GetSqlServerSPN(DataSource dataSource)`.

Reply:
This is an internal call by "GetSqlServerSPN".

Reply:
The server complete format is `protocol:name,1433`, isn't it!